blog/tests/test_blog_article_page.py

```diff
@@ -107,3 +108,62 @@ def test_that_article_with_no_recommended_articles_should_be_valid(self):
         blog_article_page.save()
 
         self.assertEqual(blog_article_page.recommended_articles, StreamValue(articles_block, []))
+
+    def test_that_altering_title_in_blog_page_model_should_not_alter_the_url_slug(self):
+        blog_article_page = self._create_blog_article_page(
+            title="Simple Article Title",
+            date=datetime(year=2019, month=5, day=1),
+            intro="Simple Article Intro",
+            read_time=5,
+        )
+        new_title = "Not So Simple Article Title"
+        expected_slug = "simple-article-title"
+
+        blog_article_page.title = new_title
+        blog_article_page.save()
+
+        self.assertEqual(blog_article_page.title, new_title)
+        self.assertEqual(blog_article_page.slug, expected_slug)
```

Comment:
Why is this added here in this PR?

Reply:
I mean additional test is not an issue, but I'm curious why is it here?

Reply:
Oh, I see you moved it. Never mind 👍

Reply:
Yup, while I was figuring out where to put new tests (integration or unit) I took a look at this test again, which was located in the integration tests file. I figured it's not a correct place for it after all, since the test clearly concerns the object alone. So I figured it I can move it while working on this file.